dozer/cogs/general.py

```diff
@@ -223,6 +225,22 @@ async def welcomeconfig(self, ctx, *, welcome_channel: discord.TextChannel):
     `{prefix}welcomeconfig #new-members` - Sets the invite channel to #new-members.
     """
 
+    @command(aliases=["setprefix"])
+    @has_permissions(manage_guild=True)
+    async def configprefix(self, ctx, prefix: str):
+        """Update a servers dynamic prefix"""
+        new_prefix = _utils.DynamicPrefixEntry(
+            guild_id=int(ctx.guild.id),
+            prefix=prefix
+        )
+        await new_prefix.update_or_add()
+        await asyncio.sleep(1)
```

Comment:
Is this needed? If so, this looks like a patch over a race condition that deserves further attention.